FWCore/MessageService/test/UnitTestClient_C.cc

```diff
@@ -23,6 +23,14 @@ namespace edmtest {
     edm::LogWarning("cat_A") << "Test of spacing:"
                              << "The following should read a b c dd:"
                              << "a" << std::setfill('+') << "b" << std::hex << "c" << std::setw(2) << "dd";
+
+    edm::LogWarning("cat_A").format("Test of format hex: {0} in hex is {0:x}", i, i);
```

Comment:
Don't you only need `i` argument since both `{}` refer to `0`?

Reply:
I thought I had only one `i` there, but apparently not.